rio_alpha/findnodata.py

```diff
@@ -133,6 +132,7 @@ def determine_nodata(src_path, user_nodata, discovery, debug, verbose):
         nodata = src.nodata
         if nodata is None:
             if discovery:
+                data = np.rollaxis(src.read(), 0, 3)
```

Comment:
@yoninachmany the `src` dataset was closed at line 125 and would need to be reopened before your new line 135 or kept open by indenting everything in-between.